src/apps/js_generic/js_generic.cpp

```diff
@@ -15,7 +16,7 @@ namespace ccfapp
   std::vector<ccf::js::FFIPlugin> get_js_plugins()
   {
 #ifdef SGX_ATTESTATION_VERIFICATION
-    return {ccf::js::openenclave_plugin};
+    return {ccf::js::openenclave_plugin, ccf::js::snp_attestation_plugin};
```

Comment:
I think `ccf::js::snp_attestation_plugin` should be set even if SGX_ATTESTATION_VERIFICATION` is false.

Reply:
We don't need a gating flag for the snp attestation, since it does not drag in external deps.